tools/DevicesApiTester/DeviceApiTester.csproj

```diff
@@ -9,6 +9,11 @@
     <Company>The .NET Foundation</Company>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- Ignore missing documentation tags (mostly already documented parameter members, anyway)-->
+    <NoWarn>1591</NoWarn>

```

Comment:
NIT: do not reset NoWarn as well in here, so just do `<NoWarn>$(NoWarn);1591</NoWarn>`